src/main/java/uk/gov/ons/ctp/response/action/service/NotificationFileCreator.java

```diff
@@ -19,51 +19,109 @@
   private static final SimpleDateFormat FILENAME_DATE_FORMAT =
       new SimpleDateFormat("ddMMyyyy_HHmm");
 
-  private final ExportFileRepository exportFileRepository;
-
   private final Clock clock;
 
+  private ActionStateService actionStateService;
+
   private PrintFileService printFileService;
 
+  private class ExportData {
+    private List<ActionRequest> ariList;
+
+    public ExportData(ActionRequest ari) {
```

Comment:
As per @insacuri, should be `ar` (or `actionRequest`)

Reply:
done